ITC205_BaseLine_Code/src/Loan.java

```diff
@@ -3,36 +3,36 @@
 import java.util.Date;
 
 @SuppressWarnings("serial")
-public class loan implements Serializable {
+public class Loan implements Serializable {
 	
-	public static enum LOAN_STATE { CURRENT, OVER_DUE, DISCHARGED };
+	public static enum LoanState { CURRENT, OVER_DUE, DISCHARGED };
 	
 	private int ID;
 	private book B;
 	private member M;
 	private Date D;
-	private LOAN_STATE state;
+	private LoanState state;
 
 	
-	public loan(int loanId, book book, member member, Date dueDate) {
+	public Loan(int loanId, book book, member member, Date dueDate) {
 		this.ID = loanId;
 		this.B = book;
 		this.M = member;
 		this.D = dueDate;
-		this.state = LOAN_STATE.CURRENT;
+		this.state = LoanState.CURRENT;
 	}
 
 	
 	public void checkOverDue() {
-		if (state == LOAN_STATE.CURRENT &&
+		if (state == LoanState.CURRENT &&
 			Calendar.getInstance().Date().after(D)) {

```

Comment:
D2.1  line 28 - fix D variable name to be more meaningful e.g 'date' or 'loanDate'